cmd/release-controller-api/http.go

```diff
@@ -1559,3 +1337,23 @@ func versionGrouping(tag string) string {
 	s := strings.Split(tag, ".")
 	return fmt.Sprintf("%s.%s", s[0], s[1])
 }
+
+func stableStream(streams []ReleaseStream) []string {
+	var stableList []string
+	for _, stream := range streams {
+		if stream.Release.Config.As == "Stable" {
```

Comment:
Better to use: `releasecontroller.ReleaseConfigModeStable`

Reply:
Done